schema_cache.rst

```diff
@@ -106,26 +111,52 @@ To refresh the cache without restarting the PostgREST server, send the server pr
      # or in docker-compose
      docker-compose kill -s SIGUSR1 <service>
 
-The above is the manual way to do it. To automate cache reloads, use a database trigger like this:
+.. _schema_reloading_notify:
+
+NOTIFY
+~~~~~~
+
+Send a `database notification <https://www.postgresql.org/docs/current/sql-notify.html>`_ from any PostgreSQL client by executing the ``NOTIFY`` command as follows:
```

Comment:
```suggestion
There are environments where you can't easily send Unix Signals(like managed containers in cloud services). For this reason, PostgREST also allows you to reload its state through PostgreSQL `NOTIFY <https://www.postgresql.org/docs/current/sql-notify.html>`_ as follows:

Reply:
I think we could add that NOTIFY can help on Windows systems too,  according to #114. 

```suggestion
There are environments where you can't send the SIGUSR1 Unix Signal (like on managed containers in cloud services or on Windows systems). For this reason, PostgREST also allows you to reload its schema cache through PostgreSQL `NOTIFY <https://www.postgresql.org/docs/current/sql-notify.html>`_ as follows:
```

Reply:
Yeah! Mentioning Windows would be great too.